utils/urls.ts

```diff
@@ -7,11 +7,16 @@ export default {
         addTree: "/admin/create",
         adminTrees: "/admin/trees",
         updateTree: (treeId: string) => `/admin/trees/${treeId}`,
+        getUser: (userId: string) => `/admin/users/${userId}`
```

Comment:
Never thought of doing this before. Neat!

Reply:
Now that I'm working on TKW-50, I'm probably going to change it back haha.